frontend/src/pages/article/ArticleBox/CardList/Card/Badge/Badge.tsx

```diff
@@ -1,14 +1,14 @@
 import * as S from "./Badge.styled";
 
 interface BadgeProps {
-  text: string;
-  bgColor?: string;
+  bgColor: string;
+  children: React.ReactNode;
```

Comment:
팀 컨벤션이 PropsWithChildren type을 사용하기로 했어서 변경하면 좋을 것 같습니다!
```suggestion
  children: React.PropsWithChildren;
```